util/memory/tracker.go

```diff
@@ -396,11 +408,66 @@ func (t *Tracker) BufferedConsume(bufferedMemSize *int64, bytes int64) {
 	}
 }
 
+// Release is used to release memory tracked, track the released memory until GC triggered if needed
+func (t *Tracker) Release(bytes int64) {
```

Comment:
I think we should add some unit test to cover the newly added function.

Reply:
Updated